src/Benchmarks.ClientJob/project.json

```diff
@@ -13,9 +13,12 @@
     "Newtonsoft.Json": "8.0.3"
   },
   "frameworks": {
-    "netstandardapp1.5": {
```

Comment:
They are supposed to be class libraries, I authored the project.json wrong.  Can you change them now?

We should also add support for NetFx 4.5, I can add this in a later change.